src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/standard.properties

```diff
@@ -380,6 +380,8 @@ doclet.ClassUse_Classes.in.0.used.by.1=Classes in {0} used by {1}
 doclet.ClassUse_PackageAnnotation=Packages with annotations of type {0}
 doclet.ClassUse_Annotation=Classes in {1} with annotations of type {0}
 doclet.ClassUse_TypeParameter=Classes in {1} with type parameters of type {0}
+doclet.ClassUse_SuperclassTypeParameter=Classes in {1} that extend types with arguments of type {0}
+doclet.ClassUse_InterfaceTypeParameter=Classes in {1} that implement types with arguments of type {0}
```

Comment:
Not sure about the wording here: I think it should be "type arguments" instead of just "arguments", and maybe "extend classes" and "implement interfaces"?

Reply:
Done. Also pulled latest master, the preview on CR is updated too.